Project.toml

```diff
@@ -12,7 +12,7 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 
 [compat]
 julia = "1"
-HTTP = "0.7, 0.8"
+HTTP = "0.7, 0.8, 0.9"
```

Comment:
Why not do this with `Downloads` instead of `HTTP`?

Reply:
> There's also the question whether we care about Julia versions older than 1.3 here. I can just switch to Downloads.jl if not.

Reply:
I think that @aviks said we didn't care about Julia versions older than 1.3, but would be good to confirm.

Reply:
Julia less than 1.3 does not work with our server implementation any more.

Reply:
Alright, I'll make the switch to Downloads then.